### PR TITLE
merge validators from unlock-app to paywall

### PR DESCRIPTION
**Pull request by cellog**
# Description

This simple PR merges the validators that will be used for validating the scrollPosition in #1744. It is a copy/paste from `unlock-app/utils/validators.js` and its test.

# Issues

Fixes #
Refs #1744 

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [X] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: paywall/src/utils/validators.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// tests whether a field's value was not entered by the user
+export const isNotEmpty = val => val || val === 0
+
+// tests whether a number is positive (0 included) and not a decimal number
```
Comment (cnasc) [inline paywall/src/utils/validators.js]: Non-negative to be pedantic 🤔

Otherwise, LG!

Comment (cellog) [reply paywall/src/utils/validators.js]: thanks, that's the verbiage I could not for the life of my find! I'll fix and merge

Comment (cellog) [reply paywall/src/utils/validators.js]: oh, I guess this is in both branches. I'll open an issue for later ;)

Comment (cellog) [reply paywall/src/utils/validators.js]: #1862

Comment (cnasc) [reply paywall/src/utils/validators.js]: No worries!

**Pull request merged**